### PR TITLE
Minor script fixes

### PR DESCRIPTION
**Pull request by s-winter**
When running an interactive shell for the idflakies user in a docker container created with iDFlakies, JAVA_HOME points to an invalid path in the container. The reason this did not affect the automated execution is that the environment is inherited from the root user when running scripts via "su - idflakies -c" and that .bashrc is not loaded for non-interactive shells (see first guard in .bashrc). To observe the issue, start an interactive shell for the idflakies user in any docker container created by the scripts and try to run maven.

- The pull request sets JAVA_HOME to the same value as for the root user's environment in order to make it work for interactive sessions (e.g., for debugging).
- As I consider it unlikely that the 'non-interactive guard' will ever be dropped from .bashrc, the pull request also removes the ineffective source instruction from run_project.sh.

### CODE REVIEW

File: scripts/docker/baseDockerfile [hunk 1]
```diff
@@ -40,7 +40,7 @@ WORKDIR /home/idflakies/
 RUN \
   wget http://mirrors.ibiblio.org/apache/maven/maven-3/3.5.4/binaries/apache-maven-3.5.4-bin.tar.gz && \
   tar -xzf apache-maven-3.5.4-bin.tar.gz && mv apache-maven-3.5.4/ apache-maven/ && \
-  echo 'export JAVA_HOME=/usr/lib/jvm/java-8-oracle' >> ~/.bashrc && \
+  echo 'export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64/' >> ~/.bashrc && \
```
Comment (winglam) [inline scripts/docker/baseDockerfile]: Isn't your change to the JAVA_HOME here specific to how your machine has Java installed in java-8-openjdk-amd64 vs. java-8-oracle?

Comment (s-winter) [reply scripts/docker/baseDockerfile]: It is specific to the setup in the docker container. Therefore, making the change in the docker file resolves the current inconsistency.

**Comment by august782** [comment]
I'm fine with merging this one. If there is nothing else from Wing, I would say merge it.

**Comment by winglam** [comment]
LGTM! Merging then.

**Pull request merged**